requirements-test.txt

```diff
@@ -2,5 +2,5 @@ psycopg2
 djangorestframework>=3.3
 coverage==3.7.1  # rq.filter: >=3,<4
 coveralls
-django-filter>=0.15,<1.0
+django-filter
```

Comment:
with experience I've learnt that restricting the versions in use is good, especially when the dependencies introduce backward incompatible changes.
This line doesn't affect the library during installation but only in travis builds, this will avoid our builds to break if a new (unsupported yet) version of django-filter intrdocues backward incompatible changes.

Therefore I would put ``django-filter>=0.15,<1.1``